packages/web/src/components/layouts/PackagePage/PackagePage.tsx

```diff
@@ -0,0 +1,596 @@
+  const [modalSortOpen, setModalSortOpen] = useState(false);
+
+  // FIXME: just for demo purpose
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+    }, 5000);
+  }, []);
+
+  const sorts = ['weight', 'popularity', 'versions'];
+
+  const loadedData = {
+    currentVersion: '20.1.0',
+    lastUpdate: '2 month ago',
+    desc: 'Copies non-react specific statics from a child component to a parent component. Similar to Object.assign, but with React static keywords blacklisted from being overridden.',
+    fullDesc:
+      'Copies non-react specific statics from a child component to a parent component. Similar to Object.assign, but with React static keywords blacklisted from being overridden. This package uses Object.defineProperty which has a broken implementation in IE8. In order to use this package in IE8, you will need a polyfill that fixes this method.',
+    usedOnList: usedOnSitesData,
+    delta: 4,
+    dependencies: ['art', 'create-react-class', 'scheduler', 'loose-envify'],
+    dependents: [
+      'ant-design-draggable-modal-4',
+      'aesthetic-react',
+      'samanage-redux-form',
+      'hoist-non-react-statics-es',
+      'react-reformed',
+      'yo-router',
+      '@canner/page-wrapper',
+      'react-falcor',
+    ],
+    keywords: [
+      '#moment',
+      '#date',
+      '#format',
+      '#time',
+      '#validate',
+      '#parse',
+      '#ender',
+      '#i18n',
+      '#l10n',
+    ],
+    modules: modulesData,
+    entries: entriesData,
+    vulnerabilities: vulnerabilitiesData,
+  };
+
+  const {
+    delta,
+    currentVersion,
+    lastUpdate,
+    desc,
+    fullDesc,
+    usedOnList,
+    modules,
+    vulnerabilities,
+    dependencies,
+    dependents,
+    entries,
+    keywords,
+  } = loadedData;
+
+  const requestSort = (sortName: string) => {
+    let direction = 'asc';
+
+    if (sortDirection === 'asc' && sortName === sortField) {
+      direction = 'desc';
+    }
+
+    setSortField(sortName);
+    setSortDirection(direction);
+  };
+
+  const closeModalHandler = () => {
+    setModalSortOpen(false);
+  };
+
+  return (
+    <>
+      <Modal isOpen={modalSortOpen} onClose={closeModalHandler}>
+        <div className={modalStyles.modalContentWrapper}>
+          <SidebarCategorySort categoryName='Sort by' returnButton={() => setModalSortOpen(false)}>
+            <div className={styles.sortButtonsMobile}>
+              {sorts.map((name) => (
+                <button
+                  key={name}
+                  type='button'
+                  className={clsx(
+                    styles.sortButtonMobile,
+                    sortField === name && styles.sortButtonMobileActive
+                  )}
+                  onClick={() => requestSort(name)}
+                >
+                  {name[0].toUpperCase() + name.slice(1)}
+
+                  {/* TODO: not sure how to handle it better right now by using <Icon> component */}
```

Comment:
You can use `currentColor` magic value in your svg, like `stroke='currentColor'` and then set color with `color` prop of `Icon`. See sprite/question-mark.svg for example.

Reply:
I get the idea and I've used it before, but in this case problem we need to change color of individual `path` inside `svg` to highlight specific arrow as active with darker color and make second one gray:
![image](https://user-images.githubusercontent.com/2999208/195047521-40c1919a-c1c1-496c-8368-9ad66c8a0397.png)
![image](https://user-images.githubusercontent.com/2999208/195047422-05d1a3a4-0795-4cfb-9126-c91bd0ddf731.png)

By using `<Icon>` and passing `color` prop will make both arrows, represented as `path`'s, use that color, showing them both as black or gray depending on direction.

Example with `<Icon kind='order' color={sortDirection === 'asc' ? '#212121' : '#DDDCE3'} />`:
![image](https://user-images.githubusercontent.com/2999208/195049174-a67d7186-234a-479e-b809-9824bee0b2c3.png)
![image](https://user-images.githubusercontent.com/2999208/195049202-17355369-eda5-48d2-bb32-4a7d2a5b4d64.png)